glide.lock

```diff
@@ -9,28 +9,99 @@ imports:
   version: 4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9
   subpackages:
   - quantile
+- name: github.com/boltdb/bolt
```

Comment:
Haha oh god, I wonder how big our binary is going to be after pulling in boltdb, cockroachdb and etcd components.  Could you let me know when you do a build how big the resulting binary is - out of curiousity?  Current production build is 27mb.